src_assets/common/assets/web/config.html

```diff
@@ -200,6 +200,19 @@ <h1 class="my-4">Configuration</h1>
           resolutions and fps are supported.<br>
           This setting does <b>not</b> change how the screen stream is sent to Moonlight
         </div>
+        <div class="mb-3">
+          <label for="max_bitrate" class="form-label">Max Bitrate</label>
+          <input
+            type="number"
+            class="form-control"
+            id="max_bitrate"
+            placeholder="5000"
+            v-model="config.max_bitrate"
+          />
+          <div class="form-text">
+            Maximum bitrate for streaming in Kbps. If not specified, the default bitrate is used
```

Comment:
```suggestion
            Maximum bitrate for streaming in Kbps.
```

We always use the default value if not specified (or if the value provided by the user is outside an acceptable range). Actually we should probably enforce a max value for the setting.